client/src/components/common/Card/index.css

```diff
@@ -13,16 +13,19 @@
   display: flex;
   padding: 0.5rem;
   align-items: center;
-  border-top-left-radius: 1rem;
-  border-bottom-left-radius: 1rem;
+  justify-content: center;
+  border-radius: 1rem;
 }
 
-.card-box__title > .title {
+.card-box__title >.title {
```

Comment:
the `>` selector is not needed in this case. I would actually got rid of the `.title` class altogether (it's quite generic name and there could be conflicts) and use `.card-box__title h5` instead.